packages/gluestick/src/generator/templates/AppServer.js

```diff
@@ -1,11 +1,23 @@
 /* @flow */
+
 import type { CreateTemplate } from '../../types';
 
 module.exports = (createTemplate: CreateTemplate) => createTemplate`
-  export default {
+/* @flow */
+
+type ApplicationServer = {
+  protocol: string;
+  host: string;
+  port: number;
+  ssetPort: number;
```

Comment:
Ok so should be https://github.com/TrueCar/gluestick/blob/f52e1748ab8b4ce3ab0a4365371e20927c554ccd/templates/new/src/config/application.server.js#L5 `assetPort`. Since `templates` are moved in `v2`, the name has been wrong.